charts/harbor-operator/values.yaml

```
@@ -23,6 +23,10 @@ imagePullSecrets: []
 # priorityClassName -- priority class to be used for the harbor-operator pods
 priorityClassName: ""
 
+postgresOperator:
+  env:
+    secrets: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
+
```

```suggestion
postgres-operator:
  configKubernetes:
    secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
```
https://github.com/zalando/postgres-operator/blob/66620d5049d807656ad3322f87e88f9a33d56859/charts/postgres-operator/values.yaml#L157